src/modules/universal-discovery/components/create/create.choose.content.type.component.js

```diff
@@ -7,19 +7,50 @@ export default class ChooseContentTypeComponent extends Component {
     constructor(props) {
         super(props);
 
+        let selected = {};
+        const isForcedContentType = props.allowedContentTypes.length === 1;
+        const allowedIncludesPreselected = props.allowedContentTypes.includes(props.preselectedContentType);
+        const isPreselectedContentType = props.preselectedContentType && (!props.allowedContentTypes.length || allowedIncludesPreselected);
+
         this._filterTimeout = null;
 
         this.updateFilterQuery = this.updateFilterQuery.bind(this);
         this.renderGroup = this.renderGroup.bind(this);
         this.renderItem = this.renderItem.bind(this);
 
+        if (isForcedContentType) {
+            selected = this.findContentType(props.allowedContentTypes[0]);
+        } else if (isPreselectedContentType) {
+            selected = this.findContentType(props.preselectedContentType);
+        }
+
+        props.onContentTypeSelected(selected);
+
         this.state = {
-            selected: {},
+            selected: selected,
             filterQuery: ''
         };
     }
 
-    updateSelectedItem(item) {
+    findContentType(identifier) {
+        let contentType = null;
+
+        Object.values(this.props.contentTypes).forEach(group => {
+            const result = group.find(contentType => contentType.identifier === identifier);
+
+            if (result) {
+                contentType = result;
+            }
+        });
+
+        return contentType;
+    }
+
+    updateSelectedItem(item, isNotSelectable) {
```

Comment:
I believe the `isNotSelectable` param should be avoided here. If you don't want to select any item, then you don't run this method.